### ISSUE TITLE
Navigation

### ISSUE THREAD

**Issue by mrtzwlkr** [corygibbons/yuki#9]
*Previous* and *Next* link would be super on permalink / single post pages

**Comment by corygibbons** [comment]
@mrtzwlkr I'm going to be doing a full rewrite of this theme fairly soon and those will be included.

**Comment by mrtzwlkr** [comment]
@corygibbons super cool!
One of the very best tumblr themes out there! Excited to see a new version coming up.

**Comment by corygibbons** [comment]
@mrtzwlkr Thanks, I appreciate it!

**Comment by PabloMontero** [comment]
+1

**Comment by corygibbons** [comment]
@mrtzwlkr Yes there will be. It's been in progress for a while but been low priority and gets pushed aside by more pressing work on my plate.

Since I don't have a timeline for the new version I'm happy to add your navigation request to the current version if that's helpful.

**Comment by mrtzwlkr** [comment]
No pressing need to do so! Just was wondering 🙂 if Yuki will still be updated. Best tumblr theme out there. Would be a shame to discontinue it 😉 

If you need any support design wise, I’m glad to help!
